orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -54,69 +42,50 @@ errorExit() {
   exit 1
 }
 
-dump_to_azure() {
+kinit_as_hdfs() {
+  # Use a new Kerberos credential cache, to keep from clobbering the default.
+  export KRB5CCNAME=/tmp/krb5cc_cloudbreak_$EUID
+  HDFS_KEYTAB=/run/cloudera-scm-agent/process/$(ls -t /run/cloudera-scm-agent/process/ | grep hdfs-NAMENODE$ | head -n 1)/hdfs.keytab
+  doLog "kinit as hdfs using Keytab: $HDFS_KEYTAB"
+  kinit -kt "$HDFS_KEYTAB" "hdfs/$(hostname -f)"
+  if [[ $? -ne 0 ]]; then
+    errorExit "Couldn't kinit as hdfs"
+  fi
+}
+
+move_backup_to_cloud () {
+  LOCAL_BACKUP="$1"
+  kinit_as_hdfs
+  doLog "INFO Uploading to ${BACKUP_LOCATION}"
+
+  hdfs dfs -mkdir -p "$BACKUP_LOCATION"
+  HDFS_FILE_LOCATION="${BACKUP_LOCATION}/${SERVICE}_backup"
```

Comment:
It's not a HDFS location. It is the location of the object-store. Please rename it accordingly.

Reply:
Changed to `OBJECT_STORE_PATH`